akka-actor-tests/src/test/scala/akka/pattern/PatternSpec.scala

```diff
@@ -52,15 +52,15 @@ class PatternSpec extends AkkaSpec {
 
   "pattern.after" must {
     "be completed successfully eventually" in {
-      // TODO after is unfortunately shadowed by ScalaTest, fix as part of #3759
+      // after is unfortunately shadowed by ScalaTest, fix as part of #3759
```

Comment:
Issue doesn't even seem to exist, or it is from some other issue tracker, so let's drop that as well
```suggestion
      // after is unfortunately shadowed by ScalaTest
```